support-frontend/assets/pages/digital-subscription-landing/components/productBlock.jsx

```diff
@@ -40,7 +40,11 @@ type DropdownPropTypes = {
 const Dropdown = ({
   children, showDropDown, product,
 }: DropdownPropTypes) => (
-  <div id={`product-details-${product}`} className={`product-block__dropdown${showDropDown ? '--show' : '--hide'}`}>
+  <div
+    id={`product-details-${product}`}
+    className={`product-block__dropdown${showDropDown ? '--show' : '--hide'}`}
```

Comment:
have you considered classnames so, if you have hide as the default state

then: 
```js
import cx from classnames

className={cx({ product-block__dropdown--show: showDropDown })}
```

Reply:
It's certainly interesting as an idea. I'll have a look and see if this might be worth abstracting. Thanks!

Reply:
Have added - thanks